eclipse-scout-core/src/layout/logicalgrid/LogicalGridLayoutConfig.ts

```diff
@@ -102,4 +120,28 @@ export class LogicalGridLayoutConfig implements LogicalGridLayoutConfigModel {
     }
     return new LogicalGridLayoutConfig(layoutConfig);
   }
+
+  static initHtmlEnvChangeHandler(widget: Widget, getter: () => LogicalGridLayoutConfig, setter: (config: LogicalGridLayoutConfig) => void) {
```

Comment:
Maybe add a little documentation? The fact that this method must be called exactly once during the initialization of the widget is not clear without reading the code.